storybook/stories/API/polar/RadialBar.stories.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { pageDataWithFillColor } from '../../data';
+import { ResponsiveContainer, RadialBarChart, RadialBar, Legend, Tooltip, PolarAngleAxis } from '../../../../src';
+import { EventHandlers } from '../props/EventHandlers';
+import { AnimationProps } from '../props/AnimationProps';
+import { legendType } from '../props/Legend';
+import { RadialBarStyle } from '../props/Styles';
+import { General as GeneralProps, Internal } from '../props/PolarComponentShared';
+import { ResponsivePropsRadialBar } from '../props/Tooltip';
+import { getStoryArgsFromArgsTypesObject } from '../props/utils';
+
+export default {
+  argTypes: {
+    ...EventHandlers,
+    ...AnimationProps,
+    legendType,
+    ...GeneralProps,
+    ...Internal,
+    ...ResponsivePropsRadialBar,
+    ...RadialBarStyle,
+    // Deprecated
+    dangerouslySetInnerHTML: { table: { category: 'Deprecated' }, hide: true, disable: true },
+    startAngle: {
+      table: { category: 'Deprecated' },
+      hide: true,
+      disable: true,
+    },
+    endAngle: {
+      table: { category: 'Deprecated' },
+      hide: true,
+      disable: true,
+    },
+    // Other
+    // baseLine: { table: { category: 'Other' } },
+    // left: { table: { category: 'Other' } },
+    // top: { table: { category: 'Other' } },
+    // xAxis: { table: { category: 'Other' } },
+    // yAxis: { table: { category: 'Other' } },
+  },
+  component: RadialBar,
+};
+
+export const API = {
+  render: (args: Record<string, any>) => {
+    // const { data } = args;
+    return (
+      <ResponsiveContainer width="100%" height={400}>
+        <RadialBarChart
+          width={400}
+          height={400}
+          innerRadius="20%"
+          outerRadius="90%"
+          data={pageDataWithFillColor.slice(0, 4)}
+          startAngle={90}
+          endAngle={-270}
+        >
```

Comment:
Nit: I would suggest every API story to be as simple as possible (not using any property that is not needed)

This would increase salience of the component in question and improve documentation clarity imho.